tests/Fixtures/TestRequest.php

```diff
@@ -19,11 +40,51 @@ class TestRequest implements RequestObjectInterface
     /**
      * Constructor
      *
+     * @param bool|null $active
+     * @param bool|null $oneTime
      * @param mixed $property
+     * @param mixed $anotherProperty
      */
-    public function __construct($property = null)
-    {
+    public function __construct(
+        ?bool $active = null,
+        ?bool $oneTime = null,
+        $property = null,
+        $anotherProperty = null
+    ) {
+        $this->active = $active;
+        $this->oneTime = $oneTime;
         $this->property = $property;
+        $this->anotherProperty = $anotherProperty;
+    }
+
+    /**
+     * Returns the bool value of this method
+     *
+     * @return bool|null
+     */
+    public function isActive(): ?bool
+    {
+        return $this->active;
+    }
+
+    /**
+     * Returns another property
+     *
+     * @return mixed
+     */
+    public function getAnotherProperty()
+    {
+        return $this->anotherProperty;
+    }
+
+    /**
+     * Returns the bool value of this method
+     *
+     * @return bool|null
+     */
+    public function isOneTime(): ?bool
+    {
+        return $this->oneTime;
```

Comment:
Methods aren't alphabetically ordered to check whether it is returning all the `gets` first then `is`.